rsconf/package_data/sirepo_sandbox/sshd_config.jinja

```diff
@@ -0,0 +1,10 @@
+AuthorizedKeysFile {{ sirepo_sandbox.guest.identity_pub_f }}
+HostKey {{ sirepo_sandbox.guest.host_key_f }}
+ListenAddress {{ sirepo_sandbox.ip }}:{{ sirepo_sandbox.ssh_port }}
+PasswordAuthentication no
+PermitTunnel no
+PermitUserEnvironment no
+PrintLastLog no
+PrintMotd no
```

Comment:
We do want this, because it contains a disclaimer. mpi_worker is different. We want users to think the are logging into a "real" machine.

Reply:
566ec33